src/wms/src/main/java/org/geoserver/wms/map/RenderedImageMapOutputFormat.java

```diff
@@ -344,8 +341,8 @@ public RenderedImageMap produceMap(final WMSMapContent mapContent, final boolean
         // TODO: handle color conversions
         // TODO: handle meta-tiling
         // TODO: how to handle timeout here? I guess we need to move it into the dispatcher?
-
         RenderedImage image = null;
+        RenderTimeStatistics statistics = new RenderTimeStatistics();
```

Comment:
Variable declared quite far away from its first usage point... is there a reason? Try to keep declaration and first usage close if at all possible.

Reply:
fixed